internal/bzlmod/go_mod.bzl

```
@@ -144,13 +144,26 @@ def _parse_directive(state, directive, tokens, comment, path, line_no):
         if len(tokens) == 3 and tokens[2][0] == ".":
```

We probably also need to check here that if a `from_version` is specified it is not being replaced with a local path
```suggestion
        if (len(tokens) == 3 and tokens[2][0] == ".") or len(tokens) > 3 and tokens[3][0] == ".":
```

---

Good catch! Fixed.